fieldmanager.php

```diff
@@ -499,6 +499,23 @@ function fm_sanitize_textarea( $value ) {
 }
 
 /**
+ * Call wp_die() with an error message for Fieldmanager_Field::_failed_validation().
+ *
+ * @since 1.1.0
+ *
+ * @param string $error_message Optional. Error message in addition to the
+ *                              default message for validation errors.
+ */
+function fm_failed_validation_to_wp_die( $error_message = '' ) {
+	wp_die( esc_html(
+		$error_message
+		. "\n\n"
+		. __( "You may be able to use your browser's back button to resolve this error.", 'fieldmanager' )
+	) );
+}
+add_action( 'fm_failed_validation', 'fm_failed_validation_to_wp_die' );
```

Comment:
I'm assuming he meant other fields of the form, because it doesn't save the rest of the fields if it dies in the middle parsing an invalid value of a certain field.